documentation-site/components/yard/config/dnd-list.ts

```diff
@@ -0,0 +1,86 @@
+import {List, arrayMove} from 'baseui/dnd-list';
+import {PropTypes} from '../const';
+import {TConfig} from '../types';
+
+const dndListConfig: TConfig = {
+  imports: {
+    'baseui/dnd-list': {
+      named: ['List', 'arrayMove'],
+    },
+  },
+  scope: {List, arrayMove},
+  theme: [],
+  props: {
+    items: {
+      value: `[
+        'Item 1',
+        'Item 2',
+        'Item 3',
+        'Item 4',
+        'Item 5',
+        'Item 6',
```

Comment:
I think 3 items are good enough to demo it and could save same vertical space here.